lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -447,6 +455,9 @@ public void OnNext(ICompletedTask completedTask)
                         _taskManager.RecordCompletedTask(completedTask);
                         TryRecovery();
                         break;
+                    case SystemState.Fail:
```

Comment:
This is not expected case. Because when we set system state to Fail, we throw exception.